tests/Driver/AmazonSqsDriverTest.php

```diff
@@ -81,9 +81,9 @@ public function testRestartBeforeStart()
         $client = $this->prepareClient('mykey1', []);
```

Comment:
Test method should be probably also renamed  from `testRestartBeforeStart` to `testShutdownBeforeStart`

Reply:
Same for redis test file.